include/net/xia_ether.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _NET_XIA_ETHER_H
+#define _NET_XIA_ETHER_H
```

Comment:
We still need `xia_ether` but only for the Netlink stuff, so the xip command can know about  `XIDTYPE_ETHER` and `struct rtnl_xia_ether_addrs`.